### DISCUSSION TITLE
Apple SSO integration

### DISCUSSION THREAD

**Discussion by adamchen** [fief-dev/fief#163] [Ideas]
Hi!

First up - thanks so much for adding the delete users function - really useful!

I couldn't help but notice that Apple SSO is not supported as part of Fief. I believe that if you were to use Fief as part of a mobile app, Apple wouldn't approve the app for iOS if other login providers (e.g. Google) exist and Apple doesn't. Is there scope to add this as a roadmap item?

Thanks!
Adam

**Comment by frankie567** [comment]
Hey! That would indeed be an important addition!

If it follows standard OAuth2, it should be quite straightforward to implement it. I'll add this to the backlog!

**Comment by adamchen** [comment]
Amazing - thank you! No further questions at this point :)
